src/components/Input.jsx

```diff
@@ -1,5 +1,19 @@
-const Input = () => {
-  return <div>Input</div>;
+const Input = ({ value, onChange, name, placeholder, classes, type }) => {
+  return (
+    <div className={`flex items-center w-1/2 text-white ${classes}`}>
+      <p className="text-xs px-1 bg-code-black absolute ml-3 mb-[3%]">{name}</p>
```

Comment:
what is the absolute for?

Reply:
the label displaces the input field
![Screenshot 2023-10-22 at 10 56 52 AM](https://github.com/acm-ucr/undecided/assets/55073341/2f64d127-023f-4227-bd52-2736f176ed2c)